Userland/Applications/Calendar/EventManager.cpp

```diff
@@ -41,21 +40,65 @@ ErrorOr<void> EventManager::save(FileSystemAccessClient::File& file)
     set_dirty(false);
 
     auto stream = file.release_stream();
-    TRY(stream->write_some(m_events.to_deprecated_string().bytes()));
+    auto json = TRY(serialize_events());
+    TRY(stream->write_some(json.to_deprecated_string().bytes()));
     stream->close();
 
     return {};
 }
 
+ErrorOr<JsonArray> EventManager::serialize_events()
+{
+    JsonArray result;
+    for (auto const& event : m_events) {
+        JsonObject object;
+        object.set("start", JsonValue(event.start.to_deprecated_string("%Y-%m-%dT%H:%M:%S"sv)));
```

Comment:
This date format gets used in several places in this file, can we make it a constant instead?